f8a_worker/workers/schemas/keywords_tagging.py

```diff
@@ -0,0 +1,32 @@
+"""JSL schema for keywords tagging worker."""
+import jsl
+from f8a_worker.schemas import JSLSchemaBaseWithRelease
+
+# Describe v1-0-0
+ROLE_v1_0_0 = "v1-0-0"
+ROLE_TITLE = jsl.roles.Var({
+    ROLE_v1_0_0: "Keywords tagging v1-0-0"
+})
+
+
+class KeywordsTaggingDetails(jsl.Document):
+    class Options(object):
+        definition_id = "keywords_tagging_details"
+        description = "Details computed by KeywordsTagging worker"
+
+    description = jsl.DictField(required=False, additional_properties=True)
+    # Keywords extracted in mercator (e.g. keywords stated in setup.py)
+    keywords = jsl.ArrayField(required=False, additional_properties=True)
```

Comment:
I'm seeing

```python
   File "/usr/lib/python3.4/site-packages/jsonschema/validators.py", line 130, in validate
     raise error
 jsonschema.exceptions.ValidationError: None is not of type 'array'

 Failed validating 'type' in schema['properties']['details']['properties']['keywords']:
     OrderedDict([('type', 'array')])

 On instance['details']['keywords']:
     None
```


If `keywords` can be `None`, then this should look like:
`keywords = jsl.OneOfField([jsl.ArrayField(required=False, additional_properties=True)], jsl.NullField())`

Reply:
Thanks! Could you please provide me ecosystem/package/version triplet for which this occurred? This is picked directly from mercator task, so having an empty array makes more sense to me from consistency POV.

Reply:
sorry, `pypi:six:1.10.0`

Reply:
Thanks, should be fixed. Nice catch 😉

Reply:
`jsl.ArrayField` doesn't seem to have a `additional_properties` argument. As I stated earlier, if `keywords` is a list of strings, then this should be defined as: `keywords = jsl.ArrayField(jsl.StringField())`

Reply:
Thanks, hopefully now :)

Reply:
Still seeing the same problem with `keywords` being `None`.
I think we need to make sure that [DataNormalizer._split_keywords()](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/data_normalizer.py#L245) returns always a list, i.e. change
```python
if keywords is None:
    return None
```
to
```python
if keywords is None:
    return []
```